<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Action</class>
 <widget class="QWidget" name="Action">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="actionLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="actionTitle">
     <property name="text">
      <string>&lt;b&gt;Action&lt;/b&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="lineTop">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutAll">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="remove">
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>❌</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="up">
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>▲</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="down">
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>▼</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layoutDefinition">
       <item>
        <layout class="QHBoxLayout" name="layoutMain"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutBaseHeader">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="toggleBaseDisplay">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>►</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="lineBaseHeader">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="layoutBaseBody" native="true">
         <layout class="QVBoxLayout" name="grpBaseBodyLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QFormLayout" name="formBase">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="formAlignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="lblSourceContext">
              <property name="text">
               <string>Source Context:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="lblLocation">
              <property name="text">
               <string>Location:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="lblStopOnFailure">
              <property name="text">
               <string>Stop On Failure:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="lblFailureEvent">
              <property name="text">
               <string>Failure Event:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="sourceContext">
              <item>
               <property name="text">
                <string>Source</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Party</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Team</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Match Team</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>All</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Enemies</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>None</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="location">
              <item>
               <property name="text">
                <string>Channel</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Instance</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Zone</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QCheckBox" name="stopOnFailure">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="lblTransformScript">
              <property name="text">
               <string>Transform Script:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="EventRef" name="failureEvent" native="true"/>
            </item>
            <item row="4" column="1">
             <widget class="ServerScript" name="transformScript" native="true"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="actionListDiv">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EventRef</class>
   <extends>QWidget</extends>
   <header>EventRef.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ServerScript</class>
   <extends>QWidget</extends>
   <header>ServerScript.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
